src/python/gennylib/parsers/cedar.py

```diff
@@ -285,6 +289,34 @@ def run(args):
 
     return metrics_file_names, my_csv2.approximate_test_run_time
 
+def is_ftdc(args):
+    return isdir(args.input_file)
+
+def run(args):
+    """
+    Runs the conversion from genny metrics to cedar format.
+    If the inputted file is a directory of FTDC data, we just
+    return those.
+
+    :param args: parsed command line args.
+    :return: list of cedar metrics file names and the approximate run time of the test
+             computed using the machine's system_time.
+
+    """
+    if is_ftdc(args):
+        time_file = os.path.join(args.input_file, "start_time.txt")
+        duration = datetime.now() - datetime.fromtimestamp(os.path.getmtime(time_file))
```

Comment:
not in love with the `start_time.txt` file since it's not extensible to other kinds of data being reported. I'd be inclined to have genny produce a "manifest" of the ftdc files and then translate that to the cedar report, but we can table that until we kill the legacy format and see how/if the cedar data evolves.